common/test/acceptance/pages/lms/courseware.py

```diff
@@ -375,6 +377,82 @@ def word_cloud_answer_list(self):
         else:
             return self.q(css='.your_words').text[0]
 
+    def is_error_message_present(self):
+        """ Check if LTI error is shown """
+        return self.q(css=".error_message").is_present()
+
+    def is_iframe_present(self):
+        """ Check if LTI iframe is present"""
+        return self.q(css=".ltiLaunchFrame").is_present()
+
+    def is_launch_url_present(self):
+        """ Check if LTI launch link is present"""
+        return self.q(css=".link_lti_new_window").is_present()
+
+    def go_to_lti_container(self):
+        """ Switch to LTI container"""
+        self.scroll_to_element('.problem-header')
+        iframe_name = self.q(css='.ltiLaunchFrame').attrs("name")[0]
+        self.browser.switch_to.frame(iframe_name)
+
+    @property
+    def switch_role_selector(self):
+        """
+        return role selector
+        """
+        self.wait_for_element_visibility(
+            '.action-preview-select',
+            'Role selector element is available'
+        )
+        return self.q(css='.action-preview-select')
+
+    def see_elem_text(self, elem_css):
+        """
+        return text of element
+        """
+        self.wait_for_element_visibility(
+            elem_css,
+            'problem score element is visible'
+        )
+        return self.q(css=elem_css).text[0]
+
+    def check_lti_component_no_elem(self, elem_css):
+        """ return LTI element bool value"""
+        return self.q(css=elem_css).is_present()
+
+
+class LTIContentIframe(CoursePage):
+
+    def is_browser_on_page(self):
+        return self.q(css=".result").present
+
+    def submit_lti_answer(self, button_css):
+        """ Click on Submit button"""
+        wait = WebDriverWait(self.browser, 10)
+        element = wait.until(EC.element_to_be_clickable((By.ID, button_css)))
```

Comment:
See if you can use [this](https://github.com/edx/bok-choy/blob/951c546f85f89e4588aec9b051d7d0304a880213/bok_choy/page_object.py#L579) instead.